.github/workflows/vci-directory-audit.yaml

```diff
@@ -0,0 +1,63 @@
+name: VCI Directory audit action
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do you still want to keep the "on push" trigger?

Reply:
I thought I had removed this, I just updated